config/routes.rb

```diff
@@ -56,6 +56,7 @@
 
           collection do
             post :mark_all_as_read, to: 'notifications#mark_as_read'
+            post :mark_message_as_read, to: 'notifications#mark_as_read'
```

Comment:
In reference to the other discussion points above, it may be a good idea to rename this route and its references to `:mark_record_as_read` so as to not specifically refer to message notifications